Terminal.Gui/Core/Application.cs

```diff
@@ -210,6 +215,24 @@ static void OnQuitKeyChanged (Key oldKey)
 		/// </summary>
 		public static bool IsMouseDisabled { get; set; }
 
+		/// <summary>
+		/// By default this is false which will continue executing the <see cref="RunLoop(RunState, bool)"/> method,
```

Comment:
Poor English phrasing. I think you mean:

"Set to true to cause the RunLoop method to exit after the first iterations. Set to false (the default) to cause the RunLoop to continue running until `Application.RequestStop()` is called."

Reply:
Done thanks.